documentation/mathmatical_operators.md

```
@@ -0,0 +1,58 @@
+# **Mathematical Operators in Python and C#**
+
+### Arithmetic Operators
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)